index.html

```diff
@@ -1,37 +1,495 @@
+                <li><a href="#">HTTP/2 Guidelines</a></li>
+                <li><a href="#">React &amp; Vue.js</a></li>
+                <li><a href="#"> Redux For Designers</a></li>
+            </ul>
+        </div>
+    </div>
+    <div class="barista-again">
+        <h2><a href="#">Smashing Newsletter</a></h2>
+        <p>Imagine you wrote a beautiful song and nobody discovered it.<br>That would make any cat sad. Don't let that happen!</p>
+        <div><a href="/the-smashing-newsletter/">
+
+
+                <img src="/smashing_magazine_clone/images/svg/cat-with-trampet.svg" alt="">
+            </a>
+        </div>
+        <div class="form-wrapper">
+            <form method="post" action="/submit" class="">
+
+
+                <div class="inline-flex padding-btn">
+                    <input type="search" name="email1" placeholder="Your(smashing)email">
+                    <button type="submit" class="btn1" id="email1" value="Submit">Subscribe <span class="arrow1">→</span>
+                        <wbr></button>
+
+                </div>
+
+            </form>
+            <small class="small">Front-end, design and UX. Sent 2× a month.<br>You can always unsubscribe with just one click.</small>
+        </div>
+
+    </div>
+    <div class="load-more1"><a class="" href="#">See All<span class="p1">Latest Articles</span></a>
+    </div>
+    <footer class="main-footer flex-container flex-column justify-content align-items bg-color">
+        <div class="bg-color">
+            <div class="footer__topics bg-color">
+                <h4 class="footer__topics__title">Browse All Smashing Magazine Topics</h4>
+                <ul class="footer__topics__list1 flex-container flex-row  align-items bg-color">
+                    <li class="footer__topics__item"><a href="/category/accessibility">Accessibility</a></li>
+                    <li class="footer__topics__item"><a href="/category/android">Android</a></li>
+                    <li class="footer__topics__item"><a href="/category/animation">Animation</a></li>
+                    <li class="footer__topics__item"><a href="/category/apps">Apps</a></li>
+                    <li class="footer__topics__item"><a href="/category/css">CSS</a></li>
+                    <li class="footer__topics__item"><a href="/category/design">Design</a></li>
+                    <li class="footer__topics__item"><a href="/category/design-patterns">Design Patterns</a></li>
+                    <li class="footer__topics__item"><a href="/category/design-systems">Design Systems</a></li>
+
+                    <li class="footer__topics__item"><a href="/category/e-commerce">E-Commerce</a></li>
+                    <li class="footer__topics__item"><a href="/category/freebies">Freebies</a></li>
+                    <li class="footer__topics__item"><a href="/category/graphics">Graphics</a></li>
+                    <li class="footer__topics__item"><a href="/category/html">HTML</a></li>
+                </ul>
```

Comment:
Two of the elements here are overflowing on the x-axis. I see that you have set the property `overflow-x: hidden;` in the body selector. Please remove that and fix the overflowing elements. It's a CSS bad practice and it might raise unnecessary errors down the road.

![Annotation 2020-06-14 094242](https://user-images.githubusercontent.com/57726348/84584591-eb1f3200-ae23-11ea-8210-e7d753f6174b.jpg)